apps/web/src/components/Notification/Type/FollowerNotification.tsx

```diff
@@ -26,6 +26,7 @@ interface FollowerNotificationProps {
   notification: NewFollowerNotification;
 }
 
+// million-ignore
```

Comment:
why this here?

Reply:
sent on discord

Reply:
This is because the styles don't work as intended when this component is optimized. When it is optimized, it becomes a block. Any components, including a <For> component can or will be wrapped in an extra `<slot>`.